src/pool/srv_pool.c

```diff
@@ -1150,6 +1152,19 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 		D_GOTO(out, rc);
 	}
 
+	uuid_generate(pool_hdl_uuid);
+	uuid_generate(cont_hdl_uuid);
```

Comment:
each time's step_up will generate a new pool hdl and cont_hdl?
So when pool svc leader switch, will cause the pool hdl cont hdl changed? how about the race condition that rebuild WIP but pool srv leader switch and IV bcast a new hdl?

Or may that if a new server pool svc step up, it generate new hdl as above; but when an existing old server pool svc step up, just reuse the existing hdl?

Reply:
yes, it would be better to use exiting uuid. I will take care of that in the following patches.